main.css

```diff
@@ -0,0 +1,43 @@
+h1{
+    font-family: monospace;
+}
+
+h2{
+    font-family: monospace;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.board{
+    position: relative;
+    width: 300px;
+    height: 300px;
+    display: flex;
```

Comment:
I think that using grid makes more sense than three flex rows within a flex column